### DISCUSSION TITLE
Forum for Support

### DISCUSSION THREAD

**Discussion by harinij** [AntlerVC/firetable#700] [Announcement]
Hi Everyone 👋

We have been using Discord for all community discussions on Rowy - including support. Which hasn't been great - as conversations tend to get lost 👎

So, we have decided to move **support** for Rowy to "Github Discussions" 🎉 It is a forum that is searchable so you can see if others faced similar issues and how they solved it. Raise your queries by starting a [new discussion](https://github.com/rowyio/rowy/discussions/new) with "**Support / Q&A**" tag. Make sure to all the relevant information so that we can investigate them.

We will continue to use [**Discord**](https://discord.com/invite/fjBugmvzZP) as a space for the community for any live discussion, general chat, or sharing of ideas and for broad topics like "how does this look in the future", "what would be the best way to contribute to X". However, every support request with specific questions would find a better place in a Github Discussion.

We hope this transition will give a better quality of support. We welcome any feedback below! 👇

We also have a new [**Testimonial**](https://github.com/rowyio/rowy/discussions/697) thread, would love for you to share how you are using Rowy/how you benefitted from it, as it helps others discover Rowy :)

Best,
Rowy Team

**Comment by EvgeniiEcorium** [comment]
I have no way to invite team members. What am I doing wrong? The version is free.

**Reply by bofeiw** [reply]
Team collaboration is available on [Rowy Pro](https://www.rowy.io/pricing).

**Comment by ghost** [comment]
I'm trying to do a Bot for whatsapp with OpenIA template, but when I click on "Deploy Automations" show this message: "the process failed" what I have to do?

**Comment by fraverdev** [comment]
It wont parse the data, any idea what im doing wrong? 

i followed the steps this guy followed. https://www.youtube.com/watch?v=zkr60T889Qc&list=RDCMUCDOUnLypYZFVGjB70QkR8ZQ&index=2

![Screenshot 2023-07-20 at 13 34 10](https://github.com/rowyio/rowy/assets/120483187/5a4eae82-c44b-495b-9dbb-a7e02dcec379)

**Reply by BeeBombshell** [reply]
@fraverdev, please check the [cloud logs](https://docs.rowy.io/cloud-logs) for any specific issues.

**Comment by earthprimeone** [comment]
i completed the install for the whatsapp general chatbot properly, entered in api keys, and setup whatsapp sender number in twilio. after that completes how do i proceed, and then test/use the app... any feedback would be helpful, as i am stuck not knowing what to do next

**Comment by mkdk2812** [comment]
I love the concept of this product, but it is not ready for prime time at all!!!  I have had projects disconnect from my DB multiple times.  I had a working project for a day then got a notice to "complete setup" of the project.  No idea how that happened.  Currently I am trying to set up a new project and the "Set Region" button is not active.  This app is super buggy!  Please fix it.

**Comment by mkdk2812** [comment]
Can someone tell me why I get a "Continue Setup" Notice on a project that is already set up?  If I click on it, it forces me to select my Firebase project that is already connected, and in the next step it says the project is already connected and I cannot go further.  What is going on with this?

<img width="997" alt="Screenshot 2023-10-29 at 9 23 05 AM" src="https://github.com/rowyio/rowy/assets/149261977/de52c496-f49f-4a92-b8b2-80ee25144620">
